specification/columns/skupricedetails.md

```diff
@@ -0,0 +1,58 @@
+# SKU Price Details
+
+The **SKU Price Details** column represents a list of relevant properties shared by all charges with the same [**SKU Price ID**](#skupriceid). These properties enable practitioners to calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics. They also help a practitioner understand the specifics of a **SKU Price ID**.
```

Comment:
These properties can provide qualitative and quantitative details about the product represented by the SKU. They may enable practitioners to calculate...